pkg/security/utils/proc_linux.go

```diff
@@ -442,6 +452,26 @@ func GetNsPids(pid uint32) ([]uint32, error) {
 	return nil, fmt.Errorf("NSpid field not found")
 }
 
+// GetPidTasks returns the task IDs of a process
+func GetPidTasks(pid uint32) ([]string, error) {
+	taskPath := procPidPath(pid, "task")
+
+	// Read the contents of the task directory
+	tasks, err := ioutil.ReadDir(taskPath)
```

Comment:
I believe ioutil is deprecated, you can use os or fs instead (not sure which one)

Reply:
Dependency removed, thanks!